src/isManager.js

```diff
@@ -1,7 +1,11 @@
 const data = require('../data/zoo_data');
 
 function isManager(id) {
-  // seu código aqui
+  // escreva sua função aqui
```

Comment:
A função já está escrita. O que falta é a implementação. O que acha de

```suggestion
  // implemente a função
```

Se fizer sentido, refatorar nos outros locais tmb.